src/3d/materials/qgsphongtexturedmaterialsettings.h

```diff
@@ -42,6 +43,13 @@ class _3D_EXPORT QgsPhongTexturedMaterialSettings : public QgsAbstractMaterialSe
      */
     QgsPhongTexturedMaterialSettings() = default;
 
+    /**
+     * Constructor for QgsPhongTexturedMaterialSettings from \a phongSettings and \a texture
```

Comment:
Please document what formats are acceptable for texture, and the ownership/lifetime requirements for those

Reply:
Done.